databrowser/src/config/builder/tourism/location.ts

```diff
@@ -185,6 +185,20 @@ export const locationTableCells = (): PropertyConfig[] => [
     class: 'w-52',
     objectMapping: { text: 'LocationInfo.RegionInfo.Name.{language}' },
   },
+
+  // FIXME: ADD FOR TESTING 533
```

Comment:
@a-crea did you forget to remove the code you added for testing or shall we remove the comment // FIXME?